### PR TITLE
search-engine-usage-study repo is now public

### PR DESCRIPTION
**Pull request by rhelmer**

### CODE REVIEW

**Review by escapewindow — CHANGES_REQUESTED** [review]
We also need to change https://github.com/mozilla-extensions/xpi-manifest/blob/main/taskcluster/ci/config.yml#L117 to be an https url rather than ssh.

File: taskcluster/ci/config.yml [hunk 1]
```diff
@@ -114,7 +114,7 @@ taskgraph:
         searchengineusagestudy:
             name: "Rally Search Engine Usage study extension"
             project-regex: searchengineusagestudys$
-            default-repository: git@github.com:mozilla-extensions/search-engine-usage-study
+            default-repository: https://github.com/mozilla-rally/search-engine-usage-study.git
```
Comment (escapewindow) [inline taskcluster/ci/config.yml:117]: I think this needs to be `mozilla-extensions`

**Comment by escapewindow** [comment]
Build is failing: circular deps, OOM, https://firefox-ci-tc.services.mozilla.com/tasks/dDf3tF4URreQh_Rra238fA/runs/0/logs/public/logs/live.log . Looks like it's failing in CI too.

**Comment by escapewindow** [comment]
Aha, we have to uplift to the release branch before this PR will go green.

**Comment by rhelmer** [comment]
> Aha, we have to uplift to the release branch before this PR will go green.

Makes sense! OK did so yesterday, let's see if it works now.

**Review by escapewindow — APPROVED** [review]

**Pull request merged**